python/hawkey/hawkeymodule.cpp

```diff
@@ -307,6 +307,7 @@ PYCOMP_MOD_INIT(_hawkey)
     PyModule_AddIntConstant(m, "FORCE_BEST", DNF_FORCE_BEST);
     PyModule_AddIntConstant(m, "VERIFY", DNF_VERIFY);
     PyModule_AddIntConstant(m, "IGNORE_WEAK_DEPS", DNF_IGNORE_WEAK_DEPS);
+    PyModule_AddIntConstant(m, "ALLOW_DOWNGRADE", DNF_ALLOW_DOWNGRADE);
```

Comment:
It could be also a negative option like do_not_allow_downgrade, but it will work only for dependencies.